app/controllers/dams_resource_controller.rb

```diff
@@ -103,6 +103,18 @@ def rdf
     params[:xsl] = "normalize.xsl"
     data = get_html_data params, nil
     render :xml => data
+  end
+  def rdf_nt
```

Comment:
Can you also add support for content-negotiation:

https://github.com/ucsdlib/damspas/blob/feature/rdf_formats/app/controllers/dams_resource_controller.rb#L67

Ideally, we should be able to either add .nt/.ttl extensions to the URL, or use an Accept header to trigger the different formats.